notebooks/data-sources/github/thoth_PR_EDA.ipynb

```diff
@@ -0,0 +1,1892 @@
+{
```

Comment:
<p>Interesting, so Bissenbay and Saisankar are not thoth members anymore, but they used to be at one point..I'm wondering if they should still count as "external" users or should be added to thoth_members list above. Any thoughts @oindrillac @suppathak?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/490/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>